src/client/Client.cc

```diff
@@ -9392,6 +9392,8 @@ int Client::_preadv_pwritev(int fd, const struct iovec *iov, unsigned iovcnt, in
 int64_t Client::_write(Fh *f, int64_t offset, uint64_t size, const char *buf,
 	                const struct iovec *iov, int iovcnt)
 {
+  uint64_t fpos = 0;
```

Comment:
no need to introduce this variable

Reply:
we need this variable to identify whether and what's the value we need to set f->pos, we set  f->pos to the new position only when offset < 0, we record the position in the variable fpos since offset would be changed later.